### ISSUE TITLE
Update Mapbox-gl-js

### ISSUE THREAD

**Issue by alex3165** [alex3165/react-mapbox-gl#53]
Update mapbox-gl-js to version 0.25

**Comment by philpl** [comment]
Can fix it this weekend together with #52

**Comment by alex3165** [comment]
Ok but apparently with the version 0.25 we should directly use the bundled file of the library so there is no point to do it ourselves anymore.

**Comment by nrako** [comment]
Hey, so yes now from version `0.25.0` Mapbox recommend to use the bundled file directly.

Trying to bundle the mapbox source with webpack will throw:
```
ERROR in ./~/mapbox-gl/js/util/worker_pool.js
Module not found: Error: a dependency to an entry point is not allowed
 @ ./~/mapbox-gl/js/util/worker_pool.js 22:30-53
```

So basically`import MapboxGl from "mapbox-gl/dist/mapbox-gl.js";`. Which works fine when the app was bundled with webpack.

However bundling an app using this package with browserify created an error because `dist/mapbox-gl.js` was parsed (i.e parsing `require("../../util/util")`), I had to use the `--noparse full_path/node_modules/mapbox-gl/dist/mapbox-gl.js` option.

That's why I thought it might be better to require `mapbox-gl` as a `peerDependency` and off-load the responsibility of bundling it with whatever bundler to the end user.

**Comment by nrako** [comment]
For reference here's my branch, couple of commits can be used as a starting point:
https://github.com/nrako/react-mapbox-gl/commits/update-mapbox-gl

**Comment by alex3165** [comment]
@nrako Thank you for your work, I have updated mapbox-gl and released it, you can use react-mapbox-gl v0.15 to use the latest version of mapbox-gl.
